packages/ffe-grid-react/src/GridRow.js

```diff
@@ -27,11 +27,18 @@ export default class GridRow extends Component {
         let content = children;
 
         const hasBackgroundColor = backgroundColors.includes(background);
+        const hasRemovedColour = removedColours.includes(background);
 
         if (hasBackgroundColor) {
             content = <div className="ffe-grid__row-wrapper">{children}</div>;
         }
 
+        if (hasRemovedColour) {
+            throw new Error(
+                `Support for the ${background} background has been removed, please see the CHANGELOG`,
```

Comment:
Please specify that it's the ffe-grid package you'll have to investigate - it might not be obvious to all users depending on context.

Reply:
I'll add the name of the component